src/main/java/de/hysky/skyblocker/skyblock/dungeon/DungeonScore.java

```diff
@@ -36,12 +39,30 @@ public static void tick() {
             }
             int score = Integer.parseInt(dungeonClearedMatcher.group("score"));
             if (!DungeonManager.isInBoss()) score += 28;
-            if (CONFIG.enableDungeonScore270 && !sent270 && score >= 270 && score < 300) {
-                MessageScheduler.INSTANCE.sendMessageAfterCooldown(Constants.PREFIX.get().getString() + CONFIG.dungeonScore270Message.replaceAll("\\[score]", "270"));
+            if (!sent270 && score >= 270 && score < 300) {
+                if (CONFIG.enableDungeonScore270Message) {
+                    MessageScheduler.INSTANCE.sendMessageAfterCooldown(Constants.PREFIX.get().getString() + CONFIG.dungeonScore270Message.replaceAll("\\[score]", "270"));
+                }
+                if (CONFIG.enableDungeonScore270Title) {
+                    client.inGameHud.setDefaultTitleFade();
+                    client.inGameHud.setTitle(Constants.PREFIX.get().append(CONFIG.dungeonScore270Message.replaceAll("\\[score]", "270")));
+                }
+                if (CONFIG.enableDungeonScore270Sound) {
+                    client.player.playSound(SoundEvents.BLOCK_NOTE_BLOCK_PLING.value(), 100f, 0.1f);
+                }
                 sent270 = true;
             }
-            if (CONFIG.enableDungeonScore300 && !sent300 && score >= 300) {
-                MessageScheduler.INSTANCE.sendMessageAfterCooldown(Constants.PREFIX.get().getString() + CONFIG.dungeonScore300Message.replaceAll("\\[score]", "300"));
+            if (!sent300 && score >= 300) {
+                if (CONFIG.enableDungeonScore300Message) {
+                    MessageScheduler.INSTANCE.sendMessageAfterCooldown(Constants.PREFIX.get().getString() + CONFIG.dungeonScore300Message.replaceAll("\\[score]", "300"));
+                }
+                if (CONFIG.enableDungeonScore300Title) {
+                    client.inGameHud.setDefaultTitleFade();
+                    client.inGameHud.setTitle(Constants.PREFIX.get().append(CONFIG.dungeonScore300Message.replaceAll("\\[score]", "300")));
```

Comment:
Is using the player's score message appropriate here, the config seems to state it being used only for chat messages. Longer score messages could also easily go off screen.

Reply:
I think it's fine. I don't want to clutter the config.